lab_1/main.py

```diff
@@ -62,10 +109,24 @@ def compare_profiles(unknown_profile: dict, profile_to_compare: dict, top_n: int
     :param top_n: a number of the most common words
     :return: the distance
     """
-    pass
+
+    if isinstance(unknown_profile, dict) and isinstance(profile_to_compare, dict):
+        if isinstance(top_n, int):
```

Comment:
There should be a check for the input arguments at the top of the function - after the documentation string. Remove nesting here and extract the check into a separate `if` block earlier.